nomos-services/data-availability/indexer/src/lib.rs

```diff
@@ -550,7 +551,10 @@ where
         let storage_adapter = DaStorage::new(storage_relay).await;
 
         let mut lifecycle_stream = service_state.lifecycle_handle.message_stream();
-        #[expect(clippy::redundant_pub_crate)]
+        #[expect(
+            clippy::redundant_pub_crate,
+            reason = "Generated by `tokio::select` macro."
+        )]
```

Comment:
I've seen a few of this now. I forgot to ask in the previous prs, why does this triggers with `tokio::select`?

Reply:
Because the macro generates a `pub(crate) enum` which then triggers the warning.